stackrox/stackrox
# ROX-17183: Migrate persistent data #1 - scanner definition

## Description

Add:
1. Large objects utilities in addition to the Gorm
2. Add migration to move scanner definition to blobstore with test.

## Checklist

If any of these don't apply, please comment below.

## Testing Performed

- Add unit test
- Run a migration test to add a scanner bundle and then upgrade and then verify.
- Run a migration without scanner bundle and upgrade and then verify.


In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.
